nimbus/src/nimbus.idl

```diff
@@ -63,6 +63,17 @@ interface NimbusClient {
     [Throws=Error]
     sequence<EnrolledExperiment> get_active_experiments();
 
+    // Getter and setter for user's participation in all experiments.
+    // Possible values are:
+    // * `true`: user has opted in.
+    // * `false`: user has opted out
+    // * null: user has not made a choice.
```

Comment:
I don't feel strongly about this, but I don't really see how the `null` result is helpful - the SDK is assuming either true or false, and there's no way for callers to know. I don't see how UI could effectively represent this tri-state - everything needs to see this as a bool, so that 3rd state just seems to offer potential confusion.

(I guess it might make sense if there was some way for the app to dictate what the default is (eg, true on nightly, false on release or similar) it might make some sense.

Reply:
This would be useful in asking if the user can opt-in to experiments. We don't want to keep asking the user to make a choice once they've made a positive or negative choice.

However, Kate's comment: 

> We have no plans to support real experiments that would require opt-in consent any time soon.

would suggest that we're not going to need such UI.

Reply:
Switched to a `true`/`false`.